non_lts_galactic/README.md

```diff
@@ -0,0 +1,12 @@
+# Non LTS ROS snap
+via [extensions](https://snapcraft.io/docs/supported-extensions), snapcraft offers support LTS versions of ROS.
```

Comment:
```suggestion
snapcraft offers support for LTS versions of ROS via [extensions](https://snapcraft.io/docs/ros-extension). They factorize some YAML code that are common to ROS snaps. While convenient, they are not necessary.
```

Reply:
Note that the updated link does not exists yet.

Reply:
Shouldn't we use the original link for the time being?

Reply:
I've edited the initial suggestion.